Impossible-Agent-Suite/contract-upgrades/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ownership-trasnfer-agent",
+  "version": "0.0.1",
+  "description": "This agent detects ownership transfers",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.0.34",
+    "bignumber.js": "^9.0.2"
```

Comment:
Please remove this dependency before merging, and update `package-lock` accordingly

Reply:
@e1Ru1o done